app/static/app/js/main.jsx

```diff
@@ -3,5 +3,17 @@ import './django/csrf';
 import React from 'react';
 import ReactDOM from 'react-dom';
 import Dashboard from './Dashboard';
+import Console from './Console';
+import $ from 'jquery';
 
-ReactDOM.render(<Dashboard/>, document.getElementById('dashboard-app'));
+$("[data-dashboard]").each(function(){
```

Comment:
Ah, now I understand the jQuery. It's not a single React app, more multiple React fragments rendered together?